### PR TITLE
fixed display of the workflows when filtering by status

### PR DESCRIPTION
**Pull request by alexnetflix**
UI does not filter recent workflows by 'RUNNING' status correctly

### CODE REVIEW

File: core/src/main/java/com/netflix/conductor/service/ExecutionService.java [hunk 1]
```diff
@@ -307,23 +309,31 @@ public void removeWorkflow(String workflowId, boolean archiveWorkflow) throws Ex
...
 				logger.error(e.getMessage(), e);
 				return null;
 			}
 		}).filter(Objects::nonNull).collect(Collectors.toList());
-		int missing = result.getResults().size() - workflows.size();
+
+		//Search again to filter out some workflows that might have been in flux state before.
+		//It's possible that during the first search ES had some workflows that were already completed in dynamo but
+		//still had a 'RUNNING' state in ES
+		SearchResult<String> repeatedResult = indexer.searchWorkflows(query, freeText, start, size, sortOptions);
```
Comment (cyzhao) [inline core/src/main/java/com/netflix/conductor/service/ExecutionService.java]: I don't think we need to search again, we just need to filter List<WorkflowSummary> workflows by RUNNING status before return the list back.

Comment (alexnetflix) [reply core/src/main/java/com/netflix/conductor/service/ExecutionService.java]: This search method is generic, meaning that the user might be searching by any status and other fields, using query or Lucene-based text search, so it's hard to extract exact search parameters to compare them with the retrieved workflow objects

Comment (cyzhao) [reply core/src/main/java/com/netflix/conductor/service/ExecutionService.java]: ah, that makes sense, didn't realize the method takes in a arbitrary query string.

Comment (cyzhao) [inline core/src/main/java/com/netflix/conductor/service/ExecutionService.java]: Thinking more this, search it repeated may not solve the problem. Let's revisit the approach.

**Pull request closed without merging**